src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineComputer.java

```diff
@@ -108,6 +113,14 @@ protected ComputeEngineCloud getCloud() {
         return node.getCloud();
     }
 
+
+
+    private Operation.Error createSnapshot(ComputeEngineCloud cloud) throws IOException, InterruptedException {
+        ComputeEngineInstance node = getNode();
+        ComputeClient client = cloud.client;
```

Comment:
No need to create a variable for this, just in-line.

Reply:
And use `getClient()` instead of field..

Reply:
Good catch.